src/differentials/composite.jl

```diff
@@ -20,13 +20,33 @@ via `comp.fieldname`.
 Any fields not explictly present in the `Tangent` are treated as being set to `ZeroTangent()`.
 To make a `Tangent` have all the fields of the primal the [`canonicalize`](@ref)
 function is provided.
+
+# Examples
+```jldoctest
+julia> Tangent{Symmetric}(; data = [1 2; 3 4])
+Tangent{Symmetric}(data = [1 2; 3 4],)
+
+julia> canonicalize(ans)
+Tangent{Symmetric}(data = [1 2; 3 4], uplo = ZeroTangent())
+
+julia> Tangent(Symmetric([1 2; 3 4]))
+Tangent{Symmetric}(data = [1 2; 3 4], uplo = 'U')
+
+julia> ChainRulesCore.construct(Symmetric, ChainRulesCore.backing(ans))
+ERROR: MethodError: no method matching Symmetric(::Matrix{Int64}, ::Char)
+```
 """
 struct Tangent{P, T} <: AbstractTangent
     # Note: If T is a Tuple/Dict, then P is also a Tuple/Dict
     # (but potentially a different one, as it doesn't contain differentials)
     backing::T
 end
 
+function Tangent(x::XT) where XT
+    kwargs = backing(x)
+    return Tangent{XT.name.wrapper}(; kwargs...)
+end
```

Comment:
`Tangent` takes the full primal type as its first argument.
Not a UnionAll over the outermost type.